modules/490-virtualization/templates/virt-controller/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
```

Comment:
where is Deployment in deployment.yaml file?

Reply:
These resources generated by virt-operator, patches for them described in kubevirt.yaml

Reply:
rename the file to vpa.yaml.

Reply:
done